code/modules/mob/living/carbon/human/human_helpers.dm

```diff
@@ -156,3 +156,14 @@
 			return FALSE
 
 	return .
+
+/mob/living/carbon/human/proc/remove_catshit()

```

Comment:
or maybe name it remove_catparts or something normal?

Reply:
dont' backseat maintain

Reply:
@optimumtact isn't it somewhat bad to have procs with obscene language though?

Reply:
there's a var ```fuck_you_dir``` in the code.
generally the code can be as rude as it wants, so long as none of it slips into in-game text.

Reply:
Also whats backseat maintaining, its not like he said "name it remove_catparts or this isn't getting merged", he made a suggestion.

Citrus plz